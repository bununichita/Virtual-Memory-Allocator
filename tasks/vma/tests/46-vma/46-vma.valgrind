==37010== Memcheck, a memory error detector
==37010== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==37010== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==37010== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==37010== Parent PID: 37009
==37010== 
==37010== 
==37010== HEAP SUMMARY:
==37010==     in use at exit: 0 bytes in 0 blocks
==37010==   total heap usage: 12 allocs, 12 frees, 8,392 bytes allocated
==37010== 
==37010== All heap blocks were freed -- no leaks are possible
==37010== 
==37010== For lists of detected and suppressed errors, rerun with: -s
==37010== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
