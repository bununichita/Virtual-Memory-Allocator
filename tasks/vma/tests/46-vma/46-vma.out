gigel
1
10
